### PR TITLE
SIP-1411

### PR DESCRIPTION
**Pull request by lauriejones**

### CODE REVIEW

File: index.scss [hunk 1]
```diff
@@ -37,7 +37,8 @@
 
 .slider-decorator-0,
 .slider-decorator-1 {
-  position: static !important;
+  //position: static !important;
```
Comment (mattbourke) [inline index.scss:40]: @lauriejones commented out code, was this missed?

**Review by mattbourke — COMMENTED** [review]
see comment on commented out code

File: index.scss (hunk 1 above)
Comment (lauriejones) [reply index.scss:40]: I believe it is intentional, but I will confirm and remove it altogether if that is the case. Thanks!

**Comment by lauriejones** [comment]
@mattbourke I didn't update the css yet?

**Pull request merged**